server/graphql/v2/mutation/AccountMutations.ts

```diff
@@ -318,7 +325,13 @@ const accountMutations = {
             throw new ValidationFailed('Invalid 2FA token');
           }
 
-          await user.update({ yubikeyDeviceId: (args.token as string).substring(0, 12) });
+          await UserTwoFactorMethod.create({
+            UserId: user.id,
+            method: TwoFactorMethod.YUBIKEY_OTP,
+            data: {
+              yubikeyDeviceId: (args.token as string).substring(0, 12),
```

Comment:
Maybe easier to type args?

Reply:
Will do! I am hoping at some point we can get automatic typing here from the mutation definition.